packages/for-ui/src/textField/TextField.tsx

```diff
@@ -117,12 +120,16 @@ export const TextField: React.ForwardRefExoticComponent<TextFieldProps> = React.
     );
 
     return (
-      <div className={fsx(['flex flex-col', className])}>
+      <div className={fsx(['flex  flex-col', className])}>
         <label>
           {label && (
             <p
               className={fsx([
-                'text-s text-shade-medium-default mb-1 font-bold antialiased',
+                'text-shade-medium-default mb-1 font-bold antialiased',
+                {
+                  large: 'text-r',
+                  medium: 'text-s',
+                }[size],
```

Comment:
すみません！ここすっかり勘違いしていたのですが、Labelなので
```suggestion
                'text-s text-shade-medium-default mb-1 font-bold',
```
でお願いします！